poker-pals/src/components/Login.vue

```diff
@@ -104,8 +104,17 @@
       });
 
       // Navigate to tables page if authentication was successful
-      this.socket.on("authenticated", userID => {
-        this.$router.push({ name: "Tables", params: {authenticated: true, userID: userID}});
+      this.socket.on("authenticatedUser", userID => {
+        this.$router.push({ name: "Tables", params: {authenticated: true, socket: this.socket, userID: userID}});
+      });
+
+      // Navigate to reports page if authentication was successful
+      this.socket.on("authenticatedAdmin", userID => {
+        this.$router.push({ name: "Reports", params: {authenticated: true, socket: this.socket, userID: userID}});
+      });
+
+      this.socket.on("banned", () => {
+        this.$router.push({ name: "Banned", params: {authenticated: true}});
```

Comment:
would being authenticated allow banned users to access the other pages?

Reply:
not quite, if the user is banned they will be able to visit the banned account page, but any attempt to visit other pages will send them right back to log in.

Reply:
How will it send them back to login? You're passing `authenticated:true` parameter

Reply:
How will it send them to log in? You're passing `authenticated:true` which is the parameter that will get checked by other pages

Reply:
because we are passing props from page to page, the boolean 'authenticated' is only as good as the previous page that sent it. if a person tries to load up tables.vue directly it will send them back to login because 'authenticated' is now false